tdrs-backend/tdpservice/users/test/test_api.py

```diff
@@ -526,6 +526,27 @@ def test_permission_delete(api_client, create_test_users):
     assert not Permission.objects.filter(codename=permission.codename).exists()
 
 
+@pytest.mark.django_db
+def test_user_no_roles_list(api_client, user):
+    """Test endpoint listing users with no roles."""
+    api_client.login(username="test__admin", password="test_password")
+
+    response = api_client.get("/v1/users/no_roles/")
+
+    groupless_users = list(
+        map(lambda u: str(u.get("id")), User.objects.filter(groups=None).values())
+    )
+    response_users = list(map(lambda u: u.get("id"), response.data))
+
+    assert all(
+        map(
+            lambda groupless_id, response_id: groupless_id == response_id,
+            groupless_users,
+            response_users,
+        )
+    )
+
```

Comment:
I think a test confirming that a user with a group does not appear in the list would be helpful